spec/integration/startup-spec.coffee

```diff
@@ -227,3 +227,13 @@ describe "Starting Atom", ->
                 [tempDirPath]
                 [otherTempDirPath]
               ].sort()
+
+  describe "opening a remote directory", ->
+    it "opens the parent directory and creates an empty text editor", ->
+      remoteDirectory = 'remote://server:3437/some/directory/path'
+      runAtom [remoteDirectory], {ATOM_HOME: atomHome}, (client) ->
+        client
+          .waitForWindowCount(1, 1000)
+          .waitForExist("atom-workspace", 5000)
+          .treeViewRootDirectories()
+          .then ({value}) -> expect(value).toEqual([remoteDirectory])
```

Comment:
I see your point, but I don't know if the asserted behavior makes sense. It's asserting that the tree view's "root directory" is `remote://something`, even though there is no such directory, and there is no directory provider for remote URLs. It seems like we've gotten the editor into a weird state here.